packages/common/core-interfaces/src/handles.ts

```diff
@@ -51,13 +51,13 @@ export interface IFluidHandleContext extends IProvideFluidHandleContext {
 /**
  * @public
  */
-export const IFluidHandle: keyof IProvideFluidHandle = "IFluidHandle";
+export const IFluidHandle = "IFluidHandle";
 
 /**
  * @public
  */
 export interface IProvideFluidHandle {
-	readonly IFluidHandle: IFluidHandle;
+	readonly [IFluidHandle]: IFluidHandle;
```

Comment:
Does this mean objects that implement `IFluidHandle` will not have an `IFluidHandle` property?

Reply:
This change has no impact on this interface, since IFluidHandle here is just a string constant of "IFluidHandle" (its just referencing the constant for it instead of inlining it). What this change does accomplish is that we keep the deduplication of this magic string (only defined once between here and IFluidHandle above) while reversing the dependency (so in the future we could make IProvideFluidHandle less public than the IFluidHandle string constant for example) and simplifying how its expressed (removing the need for using keyof.

Mainly though it seemed odd for the string constraint to depend on the interface when making the interface depend on the string constant does the same thing but with simpler syntax and just seems more logical than using keyof to define a constant string.